.changeset/popular-humans-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+Glide Core now requires PNPM 9 given PNPM 8 is no longer supported by PNPM. You'll get an error when installing this package if you have a version of PNPM older than 9 installed and have `engineStrict=true` in your `.npmrc` or `pnpm-workspace.yaml`.
```

Comment:
It won't be too long before PNPM 9 is also unsupported. But I figure we should be as accommodative as we can to consumers while still requiring a supported version.